### PR TITLE
Add recipe for new "simwrapper" package

### PR DESCRIPTION
**Pull request by billyc**
@conda-forge/help-python hello, this is my first conda package PR! 

This PR adds the package "simwrapper" (which is already on pip), to conda-forge. I am the maintainer of the pip package as well.

I used the grayskull tool to create the meta.yaml; I reviewed everything and it seems to be working and looks good to me. But as I'm new to this I'm not sure if there is more to do.

Thank you!
Billy

Checklist

- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (url) rather than a repo (e.g. git_url) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there. (I am the only maintainer! It's me, billyc)
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by billyc** [comment]
All checks passing!

@conda-forge-admin, please ping team

File: recipes/simwrapper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - python >=3.6
+  run:
+    - click
+    - python >=3.6
+
+test:
+  imports:
+    - simwrapper
+  commands:
+    - pip check
```
Comment (tdegeus) [inline recipes/simwrapper/meta.yaml:32]:
```suggestion
  commands:
    - pip check
    - simwrapper --version
```

**Review by tdegeus — COMMENTED** [review]
Not a member of the core, but I saw your request on gitter and decided to have a look.
To me everything looks great, except that you are not test if your entry point works. Usually it would have a `--help` or `--version` option that you can use for that purpose.

**Comment by billyc** [comment]
> Not a member of the core, but I saw your request on gitter and decided to have a look. To me everything looks great, except that you are not test if your entry point works. Usually it would have a `--help` or `--version` option that you can use for that purpose.

Thanks very much @tdegeus !  I've committed that change and the automated tests are running now.

**Pull request merged**